README.md

```diff
@@ -261,15 +261,8 @@ package managers:
 - `gem.txt`: run-time dependencies installed by gem.
 - `npm.txt`: run-time dependencies installed by npm.
 - `pip.txt`: a normal [pip `requirements.txt`][] file, for run-time
-  dependencies too.
-
-All of these files will install separately each line. See this example
```

Comment:
While the requirements file does install them separately, it does not tell the operator which 1 of the X addons failed to install. I added it in specific response to ODBC install issues.

But yeah I'm not worried about it TBH. I doubt people have to install ODBC often